temporalio/worker/workflow_sandbox/_runner.py

```diff
@@ -169,8 +172,13 @@ def _run_code(self, code: str, **extra_globals: Any) -> None:
             self.globals_and_locals[k] = v
         try:
             temporalio.workflow.unsafe._set_in_sandbox(True)
+            self._current_thread_id = get_ident()
```

Comment:
nit: let's do `import threading` and `threading.get_ident()` here.

Reply:
Should I do this in `_workflow_instance` as well? It's a similar situation there, just wondering if it was just something specific with the sandbox runner or a general change.

Reply:
My suggestion was general -- just an aesthetics / readability nit